packages/react-mutation-mapper/src/component/track/ExonTrack.tsx

```diff
@@ -52,12 +56,18 @@ export default class ExonTrack extends React.Component<ExonTrackProps, {}> {
             ? exonInfo.map((exon: ExonDatum, index: number) => {
                   const startCodon = exon.start;
                   const endCodon = exon.start + exon.length;
-                  const exonLength = exon.length;
-                  const isSkippable = Number.isInteger(exonLength);
-                  const stringStart = formatExonLocation(startCodon, index);
-                  const stringEnd = formatExonLocation(endCodon);
-                  const stringLength = formatExonLength(exonLength);
-
+                  const exonStartLocation = formatExonLocation(
+                      startCodon,
+                      index
+                  );
+                  const exonEndLocation = formatExonLocation(endCodon);
+                  const exonLength = formatExonLength(exon.length);
+                  const link = this.chromosome
+                      ? `https://igv.org/app/?locus=chr${this.chromosome}:${exon.genomicLocationStart}-${exon.genomicLocationEnd}&genome=hg19`
+                      : 'https://igv.org';
+                  const linkText = this.chromosome
+                      ? `hg19:chr${this.chromosome}:${exon.genomicLocationStart} - ${exon.genomicLocationEnd}`
+                      : `${exon.genomicLocationStart} - ${exon.genomicLocationEnd}`;
```

Comment:
can we add the linkout icon since it links to an external website? https://fontawesome.com/v5.15/icons/external-link-alt